tests/integration/serialization/java/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```

Comment:
This comment has little to do with this file, but why are there a bunhc of class and jar files checked in?
And do we need maven or can we rely on trivup's KAFKA_PATH?